tensorflow/python/data/benchmarks/benchmark_base.py

```diff
@@ -95,7 +115,10 @@ def run_and_report_benchmark(self,
     # Measure the per-element wall time.
     wall_time = self.run_benchmark(dataset, num_elements, iters, warmup,
                                    apply_default_optimizations)
-
+    if context.executing_eagerly():
+      name = "{}.EagerMode".format(name)
```

Comment:
replace "EagerMode" with "eager" and "GraphMode" and "graph"

Reply:
@jsimsa addressed it in the commit.